Storage/rest/structured.h

```diff
@@ -207,6 +214,8 @@ struct Structured {
             context,
             input.restful_url_prefix + '/' + kRESTfulDataURLComponent + '/' + input.field_name + '.' +
                 MatrixContainerProxy<KEY_COMPLETENESS>::PartialKeySuffix(),
+            input.restful_url_prefix + '/' + kRESTfulDataURLComponent + '/' + input.field_name + '.' +
```

Comment:
Bleah. But, it does work. Might be good to add a comment to explain the redundancy.